bumble/smp.py

```diff
@@ -553,19 +554,23 @@ def resolve(self, address):
 
 
 # -----------------------------------------------------------------------------
-class Session:
-    # Pairing methods
+class PairingMethod(enum.IntEnum):
     JUST_WORKS = 0
     NUMERIC_COMPARISON = 1
     PASSKEY = 2
     OOB = 3
 
-    PAIRING_METHOD_NAMES = {
-        JUST_WORKS: 'JUST_WORKS',
-        NUMERIC_COMPARISON: 'NUMERIC_COMPARISON',
-        PASSKEY: 'PASSKEY',
-        OOB: 'OOB',
-    }
+    def __str__(self) -> str:
+        return {
```

Comment:
`return self.name` should return the symbol name.

Reply:
Done. Invoke `.name()` when printing might be a better idea.